cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/environment/preprocessing_env.yaml

```diff
@@ -0,0 +1,19 @@
+channels:
```

Comment:
why do we have a copies of the same yml? is there a better way to avoid duplicates?

Reply:
I had discussion with Bala for this ! As of now since all the pipelines required same environment configuration, it seems feasible. But in future, we might required different env file as per the requirement.